orangecontrib/text/vectorization/document_embedder.py

```diff
@@ -87,10 +91,36 @@ def _transform(
             server_url="https://api.garaza.io",
             embedder_type="text",
         )
-        embs = embedder.embedd_data(
-            list(corpus.ngrams) if isinstance(corpus, Corpus) else corpus,
-            callback=callback,
-        )
+
```

Comment:
Do you perhaps remember that we discussed that the function should be kept as simple as possible?

Ideally, nothing would change. If something fails, there is an exception anyway.

Then, in widget's `on_expection` you can make an attempt in figuring out whether these are general internet connection errors, or something specific in our server.

Or is there a technical reason you decided against that?